src/helpers/userHelper.js

```diff
@@ -253,8 +269,8 @@ const userHelper = function () {
       `;
 
       emailSender(
-        'onecommunityglobal@gmail.com, sangam.pravah@gmail.com, onecommunityhospitality@gmail.com',
-        'Weekly Summaries for all active users...',
+        "onecommunityglobal@gmail.com, onecommunityhospitality@gmail.com",
+        "Weekly Summaries for all active users...",
```

Comment:
Neat picking: I recommend reverting to the original single quotes to maintain consistency throughout our codebase. Could we please keep the quotation style uniform unless there's a specific reason for this change?

Reply:
No specific reason--will revert to single quotes for maintaining consistency. Good catch! Thank you.